hdf5.spec

```diff
@@ -97,6 +97,14 @@ Provides:       %{name}-daos-%{daos_major} = %{version}-%{release}
 %global mpi_list %{?mpi_list} openmpi3
 %endif
 
+%if (0%{?suse_version} >= 1500)
+%global mpi_libdir %{_libdir}/mpi/gcc
+%global mpi_incldir  %{_includedir}/mpi/gcc
+%else
+%global mpi_libdir %{_libdir}
+%global mpi_incldir  %{_includedir}
+%endif
```

Comment:
Why is this necessary just to bump the Leap 15 build version to 15.2?  This seems like something that a version bump should not cause.

Reply:
It is not the version bump but SLES15.      These files should be in the mpi path and for SLES15.* this is /usr/lib64/mpi/gcc/<mpi>.